bats/tests/helpers/lima.bash

```diff
@@ -14,3 +10,12 @@ lima_home() {
         echo "${HOME}/.local/share/rancher-desktop/lima"
     fi
 }
+
+set_resources() {
+    if is_macos; then
+        _RESOURCES="/Applications/Rancher Desktop.app/Contents/Resources/resources/darwin"
+    elif is_linux; then
+        _RESOURCES="/opt/rancher-desktop/resources/resources/linux"
+    fi
+    echo "$_RESOURCES"
+}
```

Comment:
I would call this function `resources_dir`, as it returns the dir name; it doesn't set anything.

```suggestion
resources_dir() {
    if is_macos; then
        echo "/Applications/Rancher Desktop.app/Contents/Resources/resources/darwin"
    elif is_linux; then
        echo "/opt/rancher-desktop/resources/resources/linux"
    fi
}
```

Reply:
done